Source/Android/jni/AndroidCommon/IDCache.cpp

```diff
@@ -25,7 +26,9 @@ static jclass s_analytics_class;
 static jmethodID s_send_analytics_report;
 static jmethodID s_get_analytics_value;
 
-static jmethodID s_do_rumble;
+static jclass s_get_map_class;
```

Comment:
```suggestion
static jclass s_map_class;
```

Reply:
Or actually, probably `s_linked_hash_map_class`, so that nobody thinks this is referring to the `Map` interface. (Same for the static `jmethodID`s.)